thamos/lib.py

```diff
@@ -75,6 +75,7 @@
 _RETRY_ON_ERROR_COUNT = int(os.getenv("THAMOS_RETRY_ON_ERROR_COUNT", 3))
 _RETRY_ON_ERROR_SLEEP = float(os.getenv("THAMOS_RETRY_ON_ERROR_SLEEP", 3.0))
 _THAMOS_TIMEOUT = int(os.getenv("THAMOS_TIMEOUT", 2000))
+_THAMOS_TOKEN = os.getenv("THAMOS_TOKEN")
```

Comment:
isn't the token requested only by Kebechet thoth integration?

Reply:
Any user submitting requests to Thoth backend with a valid token. By providing it, an authenticated request is done - we support this feature on our client library side, that is thamos.

Reply:
Ah right, we did not filter by Integration enum, sorry, all good :)

Reply:
We need also the token for adviser right?

Reply:
Not sure what token you mean here.

Reply:
Sorry I didn't see the method changed exactly. All good!